src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -2286,7 +2286,7 @@ internal enum ErrorCode
         ERR_CollectionExpressionMissingAdd = 9215,
 
         WRN_ConvertingLock = 9216,
-        ERR_BadSpecialByRefLock = 9217,
+        // ERR_BadSpecialByRefLock = 9217,
```

Comment:
>ERR_BadSpecialByRefLock [](http://example.com/codeflow?start=11&length=23)

Is this a new error code (not officially released yet)? If so, we should be able to reuse the number. Consider marking it as available instead.